server/app/views/admin/programs/ProgramBlockAllAspectsView.java

```diff
@@ -243,19 +264,27 @@ private DivTag renderBlockList(
                   "border",
                   "border-white",
                   StyleUtils.hover("border-gray-300"),
-                  selectedClasses)
-              .with(
-                  a().withClasses("flex-grow", "overflow-hidden")
-                      .withHref(editBlockLink)
-                      .with(p(blockName), p(questionCountText).withClasses("text-sm")))
-              .with(moveButtons);
+                  selectedClasses);
 
+        String switchBlockLink =
```

Comment:
suggest to use edit instead of switch. switch implies a transition where this is just the link to the edit route.

Reply:
I renamed this to switch, because the effect of clicking it is that the view switches the block you are viewing or editing. In the case of a read only version of the view, no editing happens or is allowed and therefore "edit" is misleading.

I think there is a more general Problem with the naming of the class unrelated to this change because the view does not really display only one block, it displays all blocks and you pick one to see the details for. (goes beyond this change though)